stdlib/public/TensorFlow/Ops.swift

```diff
@@ -577,6 +577,41 @@ public extension Tensor where Scalar : FloatingPoint & Equatable {
   }
 }
 
+public extension Tensor where Scalar : FloatingPoint {
```

Comment:
Please constrain Scalar on TensorFlowFloatingPoint to be consistent with other extensions. Once that’s done, you should be able to drop all where clauses from the @differentiable attributes you added. This simplified everything.

Reply:
Thanks done!